TypeCal.jacc

```diff
@@ -44,12 +45,16 @@ declaration: REAL ID
 { $$ = new DeclarationNode((String) $2, Type.REAL); }
 | INT ID
 { $$ = new DeclarationNode((String) $2, Type.INTEGER); }
-| ID ID
+;
+
+record_declaration: ID ID
 { $$ = new RecordDeclarationNode((String)$1, (String)$2); }
 ;
 
 declaration_list: declaration_list SEMI declaration
 { $$ = addDeclaration($1, (DeclarationNode)$3); }
+| declaration_list SEMI record_declaration
```

Comment:
Make sure record_declaration is still accounted for now that it's not part of declaration anymore.